infra/spawn_data_pipeline.py

```diff
@@ -75,37 +75,36 @@ def main():
     with open(args.data_pipeline_json) as pipeline_file:
         raw_tasks = yaml.safe_load(pipeline_file.read())
 
-    for task in raw_tasks["tasks"]:
```

Comment:
Unrelated to this PR, but we should really add some tests for this module (mocking Taskcluster), as otherwise we notice breakages really late

Reply:
Could you file an issue for this?

Reply:
Done https://github.com/mozilla/bugbug/issues/613